miner/worker.go

```diff
@@ -1167,7 +1169,10 @@ loop:
 			// Reorg notification data race between the transaction pool and miner, skip account =
 			log.Trace("Skipping account with hight nonce", "sender", from, "nonce", tx.Nonce())
 			txs.Pop()
-
+		case errors.As(err, &errL1):
```

Comment:
Let's add some unit tests.

Reply:
added test that checks if tx retried in next block